script.py

```diff
@@ -1,13 +1,38 @@
 import os
-from datetime import datetime
+import requests
 
 # Get input from environment variables
+website = os.getenv("INPUT_WEBSITE", "")
+test_name = os.getenv("INPUT_TEST_NAME", "")
+cj_token = os.getenv("INPUT_CJ_TOKEN", "")
 instructions = os.getenv("INPUT_INSTRUCTIONS", "")
 
-# Perform some processing
-output = f"Input received: {instructions}. Current time is {datetime.now()}."
+def run():
+    if not (website and test_name and cj_token and instructions):
+        return "Failed: All required fields were not provided."
+
+    response = requests.post(
+        "https://cj-backend.foreai.co/test-case/",
+        headers={
+            "Authorization": f"Bearer {cj_token}"},
+        json = {
+            "website": website,
+            "name": test_name,
+            "description": instructions,
+        },
+        headers = {
+            "Authorization": f"Bearer {cj_token}",
+            "Content-Type": "application/json"
+        })
+
+    if response.status_code == 201:
+        # Success: The test case was created successfully
+        return f"Test successfully generated. Visit: https://cj.foreai.co/{response.text}"
+    else:
+        return f"Failed to create test case: {response.json()}"
```

Comment:
maybe we can figure out how to make a red X in the action. OTOH, we may not want to fail the entire flow just b/c our backend is down or broken ?

Reply:
Yes, seems like sys.exit is the way to go. Will test this now.